blinkpy/camera.py

```diff
@@ -246,14 +248,14 @@ def extract_config_info(self, config):
         self.serial = config.get("serial")
         self._version = config.get("fw_version")
         self.motion_enabled = config.get("enabled", "unknown")
+        self.battery_voltage = config.get("battery_voltage", None)
```

Comment:
_battery_voltage

Reply:
The HA project has strict typing and lint rules - while it "looks" like you had a property, it's not allowed to be accessed as it's "private" according to the linting rules.  In order for a project like HA to access the variable we need the property defined... It's all a bit confusing for sure (and I'm no expert).

Reply:
Oh, I see, yea, the `@property` syntax sugar. Gotcha. So, you're saying the battery_voltage is "important" (common) enough that you want the `@property` accessor to make it easy to pull it into HA directly (without parsing all the json), but other fields such as `motion_enabled` or `wifi_strength` aren't as common.

I'll update the PR....